tests/test_astrometrySourceSelector.py

```diff
@@ -96,10 +99,27 @@ def testSelectSources_bad(self):
             self.assertNotIn(x, result.sourceCat['id'], "should not have found %s" % badFlags[i])
 
     def testSelectSources_bad_centroid(self):
+        """Unflagged NaN centroids should just assert."""
         add_good_source(self.src, 1)
         self.src[0].set('slot_Centroid_x', np.nan)
+        with self.assertRaises(AssertionError):
+            self.sourceSelector.selectSources(self.src)
+        self.src[0].set('slot_Centroid_x', 5.0)
+        self.src[0].set('slot_Centroid_y', np.nan)
+        with self.assertRaises(AssertionError):
+            self.sourceSelector.selectSources(self.src)
+
+    def testSelectSources_bad_centroid_Sigma(self):
+        """Reject sources with NaN centroid_[xy]Sigma.
+        Currently, there is no guarantee that a valid centroid_flag implies
+        a finite variance."""
+        add_good_source(self.src, 1)
+        add_good_source(self.src, 2)
```

Comment:
More snake case.  Looks like a lot of it predates these changes; maybe file a ticket to fix that later?